### PR TITLE
Update to Zivid SDK v2.4.0

### PR DESCRIPTION
**Pull request by SatjaSivcev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,43 +50,57 @@ Samples indicated by **(cpp)** are written in C++, samples indiced by **(csharp)
...
-[csharp-captureViaGenICam-url]: source/csharp/Camera/Basic/CaptureViaGenICam
-[csharp-captureViaZivid-url]: source/csharp/Camera/Basic/CaptureViaZivid
\ No newline at end of file
+[hdev-get_zivid_sample_data_dir-url]: source/hdev/Procedures/get_zivid_sample_data_dir.hdvp
+
+
+[halcon-url]: https://www.mvtec.com/products/halcon/
+
+[knowladge-base-url]: https://support.zivid.com/
+[zivid-software-installation-url]: https://support.zivid.com/latest/rst/academy/getting-started/zivid-software-installation.html
```
Comment (chrisasc) [inline README.md:99]: remove rst from these as well

**Review by chrisasc — COMMENTED** [review]
remove rst from these links too

File: README.md (hunk 1 above)
Comment (SatjaSivcev) [reply README.md]: done

**Review by chrisasc — APPROVED** [review]
lgtm, but we should ideally check it before we merge

**Pull request merged**